server/routes/referrals/referralStartView.ts

```diff
@@ -1,7 +1,13 @@
 import DraftReferralsListPresenter from './draftReferralsListPresenter'
+import { FormValidationError } from '../../utils/formValidationError'
+import ReferralDataPresenterUtils from './referralDataPresenterUtils'
+import viewUtils from '../../utils/viewUtils'
 
 export default class ReferralStartView {
-  constructor(private readonly presenter: DraftReferralsListPresenter) {}
+  constructor(
+    private readonly presenter: DraftReferralsListPresenter,
+    private readonly error: FormValidationError | null = null
```

Comment:
This is inconsistent with the way we've built our other views - the idea is that the presenter should be the only source of data for the view. The presenter would [take a `FormValidationError` in its constructor](https://github.com/ministryofjustice/hmpps-interventions-ui/blob/490a53623cc0d1504f0c2689b50dffefb8655c09/server/routes/referrals/needsAndRequirementsPresenter.ts#L8), and then expose any error information to the view via some sort of [error message property](https://github.com/ministryofjustice/hmpps-interventions-ui/blob/490a53623cc0d1504f0c2689b50dffefb8655c09/server/routes/referrals/needsAndRequirementsPresenter.ts#L26). Can we keep it consistent?

Reply:
Have re-worked this to use our usual pattern.